telegram/ext/_applicationbuilder.py

```diff
@@ -487,6 +487,11 @@ def proxy_url(self: BuilderType, proxy_url: str) -> BuilderType:
         """Sets the proxy for the :paramref:`~telegram.request.HTTPXRequest.proxy_url`
         parameter of :attr:`telegram.Bot.request`. Defaults to :obj:`None`.
 
+        Note:
+            this is not the proxy used for all http requests.
```

Comment:
```suggestion
```
I don't think this note is necessary because of the 'see also' you added below

Reply:
As a user personally I think it's necessary to note this...

Reply:
hinrich also voted to remove this note (https://github.com/python-telegram-bot/python-telegram-bot/pull/3816#issuecomment-1654517762)